circle.yml

```diff
@@ -18,6 +41,8 @@ test:
     - mysql -u ubuntu -e 'CREATE DATABASE circle_test;'
     - psql -U ubuntu -c 'DROP DATABASE circle_test;'
     - psql -U ubuntu -c 'CREATE DATABASE circle_test;'
+    # Run unit tests on Oracle
+    - npm run test-oracle-ci
```

Comment:
Should be run in docker container (like other tests). We need to pass env variables specified in package.json.